rs/tests/node/BUILD.bazel

```diff
@@ -70,3 +70,35 @@ system_test(
         "@crate_index//:slog",
     ],
 )
+
+# bazel test --test_output=streamed //rs/tests/node:launch_single_guest --test_timeout=3600 --test_arg=--debug-keepalive
+system_test(
+    name = "launch_single_guest",
+    tags = [
+        "manual",
```

Comment:
Shouldn't these be tagged as `dynamic_testnet` as well such that you can launch them using
```
ict testnet create launch_single_guest ...
```

Reply:
Cool! Done.